src/app/lecture/knowledge-node/learning-objects/question/question.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, OnInit } from '@angular/core';
+import { LearningObjectComponent } from '../learning-object-component';
+import { Question } from './model/question.model';
+import { QuestionService } from './service/question.service';
+
+@Component({
+  selector: 'cc-question',
+  templateUrl: './question.component.html',
+  styleUrls: ['./question.component.css']
+})
+export class QuestionComponent implements OnInit, LearningObjectComponent {
+
+  data: Question;
```

Comment:
I would rename this to "question". While its clear at this point since we have a few lines of code, imagine what would happen if this component grew and there were a lot of things that could be "data".

Reply:
The data field "implements" data from LearningObjectComponent interface and is used in LearningObjectContainerComponent to pass data to a concrete dynamically loaded LO component.

Reply:
Got it. Then it's good to go.